presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -283,4 +287,32 @@ public QueryManagerConfig setQueryExecutionPolicy(String queryExecutionPolicy)
         this.queryExecutionPolicy = queryExecutionPolicy;
         return this;
     }
+
+    @Min(1)
+    public int getInitialMinWorkersRequired()
+    {
+        return initialMinWorkersRequired;
+    }
+
+    @Config("query-manager.initial-min-workers-required")
```

Comment:
Thought about this quite a bit and discussed @electrum. Let's call this initializationRequiredWorkers. Make the change to the property name also.